index.html

```diff
@@ -34,27 +34,54 @@
   </head>
 
   <body>
-    <div id="header_main" style="display: block;">
+    <div id="header_main" style="display: block;" class="mb-0 pb-0">
       <canvas></canvas>
       <div id="overlay">
         <div id="header_window">
             <div id="header">
                 <img src="https://learning-at-home.github.io/logo.png" id="bug-logo"
                     style="width: 40%; max-height: 320px; max-width: 320px; z-index:1000; position: relative;">
                 <br>
-                <h1 class="faded title title_elem" style="margin-top:-5%;">
-                    <p style="margin-top: 0px; margin-bottom:0px;">
+                <h1 class="faded title title_elem mb-1 pb-1" style="margin-top:-40px; margin-bottom:-10px">
+                    <p style="margin-top: 0px; font-weight:bolder; margin-bottom:0px;">
                         <span id="title_text">Training Transformers Together</span>
                     </p>
-                    <p style="font-size: 18px; margin-top:0px; margin-bottom:0px;">
-                        large-scale deep learning for everyone,&nbsp;by&nbsp;everyone</p>
+                    <p style="font-size: 18px; margin-top:0px; margin-bottom:5px;">
+                        large-scale deep learning for everyone, by everyone</p>
+                    <p style="font-size: 18px; font-weight:lighter; margin-top:0px; margin-bottom:0px;">
+                        A NeurIPS 2021 Demonstration</p>
                 </h1>
             </div>
         </div>
       </div>
   </div>
   <script src="./header-animate.js"></script>
 
+  <div class="container d-flex justify-content-center mb-2 pb-2" style="width:500px">
+          <div class="row text-center align-items-center justify-content-center">
+              <div class="col-3">
+                  <a href="https://research.yandex.com/">
+                  <img src="logos/yandex.png" class="img-fluid center-block" style="max-width: 75%" alt="Yandex Research">
+                  </a>
+              </div>
+              <div class="col-3 px-2">
+                  <a href="https://huggingface.co/">
+                  <img src="logos/huggingface.png" class="img-fluid center-block" style="max-width: 75%" alt="Hugging Face">
+                  </a>
+              </div>
+              <div class="col-3 px-3">
+                  <a href="https://www.hse.ru/en/">
+                  <img src="logos/hse.png" class="img-fluid center-block" style="max-width: 75%" alt="HSE University">
+                  </a>
+              </div>
+              <div class="col-3 px-2">
+                  <a href="http://www.washington.edu/">
+                  <img src="logos/uwash.svg" class="img-fluid center-block" style="max-width: 75%" alt="University of Washington">
```

Comment:
Can you take the logo from here: https://www.washington.edu/brand/graphic-elements/logo-guidelines/. If png works, then https://s3-us-west-2.amazonaws.com/uw-s3-cdn/wp-content/uploads/sites/98/2014/09/07212706/W-Logo_RegistrationMark_Purple_26851.png is best.